datasets/xtreme/README.md

```diff
@@ -1,8 +1,467 @@
+  - ru
+  PAN-X.sw:
+  - sw
+  PAN-X.ta:
+  - ta
+  PAN-X.te:
+  - te
+  PAN-X.th:
+  - th
+  PAN-X.tl:
+  - tl
+  PAN-X.tr:
+  - tr
+  PAN-X.ur:
+  - ur
+  PAN-X.vi:
+  - vi
+  PAN-X.yo:
+  - yo
+  PAN-X.zh:
+  - zh
+  XQuAD:
+  - ar
+  - de
+  - vi
+  - zh
+  - en
+  - es
+  - hi
+  - el
+  - ru
+  - th
+  - tr
+  MLQA.ar.ar:
+  - ar
+  MLQA.ar.de:
+  - ar
+  - de
+  MLQA.ar.vi:
+  - ar
+  - vi
+  MLQA.ar.zh:
+  - ar
+  - zh
+  MLQA.ar.en:
+  - ar
+  - en
+  MLQA.ar.es:
+  - ar
+  - es
+  MLQA.ar.hi:
+  - ar
+  - hi
+  MLQA.de.ar:
+  - de
+  - ar
+  MLQA.de.de:
+  - de
+  MLQA.de.vi:
+  - de
+  - vi
+  MLQA.de.zh:
+  - de
+  - zh
+  MLQA.de.en:
+  - de
+  - en
+  MLQA.de.es:
+  - de
+  - es
+  MLQA.de.hi:
+  - de
+  - hi
+  MLQA.vi.ar:
+  - vi
+  - ar
+  MLQA.vi.de:
+  - vi
+  - de
+  MLQA.vi.vi:
+  - vi
+  MLQA.vi.zh:
+  - vi
+  - zh
+  MLQA.vi.en:
+  - vi
+  - en
+  MLQA.vi.es:
+  - vi
+  - es
+  MLQA.vi.hi:
+  - vi
+  - hi
+  MLQA.zh.ar:
+  - zh
+  - ar
+  MLQA.zh.de:
+  - zh
+  - de
+  MLQA.zh.vi:
+  - zh
+  - vi
+  MLQA.zh.zh:
+  - zh
+  MLQA.zh.en:
+  - zh
+  - en
+  MLQA.zh.es:
+  - zh
+  - es
+  MLQA.zh.hi:
+  - zh
+  - hi
+  MLQA.en.ar:
+  - en
+  - ar
+  MLQA.en.de:
+  - en
+  - de
+  MLQA.en.vi:
+  - en
+  - vi
+  MLQA.en.zh:
+  - en
+  - zh
+  MLQA.en.en:
+  - en
+  MLQA.en.es:
+  - en
+  - es
+  MLQA.en.hi:
+  - en
+  - hi
+  MLQA.es.ar:
+  - es
+  - ar
+  MLQA.es.de:
+  - es
+  - de
+  MLQA.es.vi:
+  - es
+  - vi
+  MLQA.es.zh:
+  - es
+  - zh
+  MLQA.es.en:
+  - es
+  - en
+  MLQA.es.es:
+  - es
+  MLQA.es.hi:
+  - es
+  - hi
+  MLQA.hi.ar:
+  - hi
+  - ar
+  MLQA.hi.de:
+  - hi
+  - de
+  MLQA.hi.vi:
+  - hi
+  - vi
+  MLQA.hi.zh:
+  - hi
+  - zh
+  MLQA.hi.en:
+  - hi
+  - en
+  MLQA.hi.es:
+  - hi
+  - es
+  MLQA.hi.hi:
+  - hi
+  tydiqa:
+  - fi
+  - te
+  - ru
+  - ar
+  - id
+  - en
+  - sw
+  - ko
+  - bn
+  bucc18.de:
+  - de
+  bucc18.fr:
+  - fr
+  bucc18.zh:
+  - zh
+  bucc18.ru:
+  - ru
+  tatoeba.afr:
+  - af
+  tatoeba.ara:
+  - ar
+  tatoeba.ben:
+  - bn
+  tatoeba.bul:
+  - bg
+  tatoeba.deu:
+  - de
+  tatoeba.cmn:
+  - zh
+  tatoeba.ell:
+  - el
+  tatoeba.est:
+  - et
+  tatoeba.eus:
+  - eu
+  tatoeba.fin:
+  - fi
+  tatoeba.fra:
+  - fr
+  tatoeba.heb:
+  - he
+  tatoeba.hin:
+  - hi
+  tatoeba.hun:
+  - hu
+  tatoeba.ind:
+  - id
+  tatoeba.ita:
+  - it
+  tatoeba.jav:
+  - jv
+  tatoeba.jpn:
+  - ja
+  tatoeba.kat:
+  - ka
+  tatoeba.kaz:
+  - kk
+  tatoeba.kor:
+  - ko
+  tatoeba.mal:
+  - ml
+  tatoeba.mar:
+  - mr
+  tatoeba.nld:
+  - nl
+  tatoeba.pes:
+  - fa-IR
+  tatoeba.por:
+  - pt
+  tatoeba.rus:
+  - ru
+  tatoeba.spa:
+  - es
+  tatoeba.swh:
+  - sw
+  tatoeba.tam:
+  - ta
+  tatoeba.tel:
+  - te
+  tatoeba.tgl:
+  - tl
+  tatoeba.tha:
+  - th
+  tatoeba.tur:
+  - tr
+  tatoeba.urd:
+  - ur
+  tatoeba.vie:
+  - vi
+  SQuAD:
+  - en
+licenses:
+- apache-2.0
+- cc-by-4.0
+- cc-by-2.0
+- cc-by-sa-4.0
+- other-Licence Universal Dependencies v2.5
+- other-Attribution-NonCommercial 4.0 International
```

Comment:
I believe this is `cc-by-nc-4.0` (Creative Commons Attribution Non Commercial 4.0 International) ?
```suggestion
- cc-by-nc-4.0
```

Reply:
This is the license of the [XNLI](https://paperswithcode.com/dataset/xnli) dataset, and yes, you are right.

Reply:
Ok cool ! Let's merge now :D